src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesEdit.js

```diff
@@ -107,27 +116,48 @@ export default class ExternalDataSourcesEdit extends React.Component {
               dataOptions={
                 [{ value: '1', label: 'Package' }]
               }
+              disabled={isReadonly}
               label={<FormattedMessage id="ui-local-kb-admin.settings.externalDataSources.recordType" />}
               name={`${name}.rectype`}
-              required
+              required={!isReadonly}
```

Comment:
It's still required, right? Just not editable?

Reply:
I removed this in the readonly case just because LOCAL doesn't have a rectype, which stopped you from saving changes made to it... and Owen mentioned that as of right now he didn't have a use case for any other readonly externalSources, so I thought this was sufficient for now.